src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java

```diff
@@ -85,8 +86,8 @@ public void initialize() {
         .putValue("contextValues", new HashMap<String, Object>())
         .putValue("productIdentifier", "id")
         .putValue("adobeVerboseLogging", true),
-      analytics,
-      Logger.with(VERBOSE),
+        analytics,
+        Logger.with(VERBOSE),
         mockHeartbeatFactory);
```

Comment:
Shouldn't `analytics`, `logger.with(VERBOSE)` and `mockHeartbeatFactory` all share a level of indentation that's one less that all the `.putValue`s?